### ISSUE TITLE
Training resnets on a loop results in loss plots with multiple lines

### ISSUE THREAD

**Issue by nburner96** [amaiya/ktrain#527]
I am training multiple resnet models on a loop, each with a different set of training and testing images. Each iteration begins with `tf.keras.backend.clear_session()`. When I call the following code, on initial iterations the loss plot looks normal, however on later ones multiple lines are displayed:

```
        plt = learner.plot(return_fig =  True)
        plt.savefig(f'{model_path}/loss.tiff')
```

![image](https://github.com/amaiya/ktrain/assets/21323696/c481b235-bd60-4f4e-886a-5c255f662ee3)

Do you know what could be causing that?

**Comment by amaiya** [comment]
It's hard to tell, since you didn't share any code reproducing the problem.  You can try looking at `Learner.history.history`  (which is a Python dictionary) to examine what's happening (and possibly edit as needed when plotting).
